edit_patient_registration_details.md

```diff
@@ -0,0 +1,23 @@
+###Editing  Patient Registration
```

Comment:
@solemabrothers instead of creating a new page add it as a section to the Register Patient page

Reply:
Done